<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Equitux</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="button_calculate">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calculate EQ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="board_0">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Flop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="board_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="board_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="board_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="board_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="win_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="range_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>3700</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="win_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="win_1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="win_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="win_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="range_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>3700</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Win</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="win_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hand range</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="range_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>3700</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="range_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>3700</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="range_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>3700</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="range_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>3700</number>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear_0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cancel_32x32_round.png</normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cancel_32x32_round.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Turn</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>River</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_range_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cards_yellow_32x32.png</normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cards_yellow_32x32.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="select_range_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cards_yellow_32x32.png</normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cards_yellow_32x32.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="select_range_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cards_yellow_32x32.png</normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cards_yellow_32x32.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="select_range_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cards_yellow_32x32.png</normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cards_yellow_32x32.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="select_range_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cards_yellow_32x32.png</normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cards_yellow_32x32.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="select_range_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cards_yellow_32x32.png</normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cards_yellow_32x32.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cancel_32x32_round.png</normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cancel_32x32_round.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cancel_32x32_round.png</normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cancel_32x32_round.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cancel_32x32_round.png</normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cancel_32x32_round.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cancel_32x32_round.png</normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cancel_32x32_round.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cancel_32x32_round.png</normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cancel_32x32_round.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cancel_32x32_round.png</normaloff>../../../../../../../media/datos/documentos/programas/qt/equiprueba/equiprueba/styles/icon_cancel_32x32_round.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
